packages/code-du-travail-frontend/src/modules/besoin-plus-informations/index.tsx

```diff
@@ -118,6 +118,7 @@ export const BesoinPlusInformations = () => {
             target="_blank"
             data-testid="result-search-service"
             ref={setLinkRef}
+            title={`${result.url}`}
```

Comment:
```suggestion
            title="Consulter le site des renseignements du département XX"
```

XX est remplacé par la saisie de l'utilisateur :)